Sail/src/Sail/entities/systems/Audio/AudioSystem.cpp

```diff
@@ -536,4 +536,10 @@ bool AudioSystem::onEvent(const Event& event) {
 	}
 
 	return true;
-}
\ No newline at end of file
+}
+
+#ifdef DEVELOPMENT
+unsigned int AudioSystem::getByteSize() const {
+	return BaseComponentSystem::getByteSize() + sizeof(*this);
```

Comment:
Also need to include AudioEngine size since that's created by the AudioSystem

Reply:
Fixed, thanks for the reminder